pinot-plugins/pinot-stream-ingestion/pinot-pulsar/pom.xml

```diff
@@ -37,22 +37,22 @@
   <properties>
     <phase.prop>package</phase.prop>
     <pinot.root>${basedir}/../../..</pinot.root>
-    <pulsar.version>2.7.2</pulsar.version>
-    <jetty-server.version>9.4.45.v20220203</jetty-server.version>
+    <pulsar.version>2.10.0</pulsar.version>
```

Comment:
Here, again we might need to check the tests because I remember them failing with the current localstack version.